### ISSUE TITLE
Feature request: Share cache with requests-cache

### ISSUE THREAD

**Issue by franz101** [requests-cache/aiohttp-client-cache#157]
### Use case
Use the same SQLite backend

### Workarounds
Have the same key generation

### Plan to implement
Maybe

```python
from aiohttp_client_cache import CachedSession, SQLiteBackend
import requests_cache
cache = "drive/MyDrive/xyz.sqlite"
url = 'https://example.com'

requests_cache.install_cache(cache)
requests.get(url)

async def main():
  global url
  async with CachedSession(cache=SQLiteBackend(cache)) as session:  
    r = await session.get(url)
    print(r.from_cache)
await main()
```

This evaluates to `False`, but should be `True`

**Comment by JWCook** [comment]
I agree that this would be nice to have. Unfortunately this would be quite a bit more complicated than changing the cache keys. Problems include:
1. requests-cache uses a different serialization format using the [cattrs](https://github.com/python-attrs/cattrs) library
2. [`aiohttp.ClientResponse`](https://github.com/aio-libs/aiohttp/blob/7911f1e9542afd0623b61f75c5e367c3c0e185bd/aiohttp/client_reqrep.py#L673) response objects look quite different from [`requests.Response`](https://github.com/psf/requests/blob/cdbc2e271529f467b278b2760f12ee0b5d6930d3/requests/models.py#L641) objects
3. Objects serialized with `pickle` are tied to a specific class, and will throw an error if the original serialized class isn't available

A possible solution might look something like:
* Update this library to use cattrs (to "unstructure" a response object into primitive types before pickling)
* Make a new response class compatible with both aiohttp and requests
    * Or add handling for missing attributes unique to each HTTP library
* Add all (or most) of requests-cache's request matching behavior to this library so cache keys will be the same

There could be a simpler solution, but I can't think of one right now.
